src/Foundatio.Extensions.Hosting/Startup/StartupActionsContext.cs

```diff
@@ -28,27 +28,27 @@ internal void MarkStartupComplete(RunStartupActionsResult result)
     public async Task<RunStartupActionsResult> WaitForStartupAsync(CancellationToken cancellationToken, TimeSpan? maxTimeToWait = null)
     {
         bool isFirstWaiter = Interlocked.Increment(ref _waitCount) == 1;
-        var startTime = SystemClock.UtcNow;
-        var lastStatus = SystemClock.UtcNow;
+        var startTime = DateTime.UtcNow;
+        var lastStatus = DateTime.UtcNow;
         maxTimeToWait ??= TimeSpan.FromMinutes(5);
 
-        while (!cancellationToken.IsCancellationRequested && SystemClock.UtcNow.Subtract(startTime) < maxTimeToWait)
+        while (!cancellationToken.IsCancellationRequested && DateTime.Now.Subtract(startTime) < maxTimeToWait)
```

Comment:
You are comparing now to utc time, I think all of this logic should be in utc to count for day light savings